### PR TITLE
Fix Date on connexion

### PR DESCRIPTION
**Pull request by Drosaca**

### CODE REVIEW

**Comment by jess-sys** [comment]
https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Global_Objects/Date/parse

File: src/classes/Standardizer/plugins/Discord/getters/getConnections.ts [hunk 1]
```diff
@@ -24,7 +24,7 @@ Discord.prototype.getConnections = withAutoParser(async parser => {
           },
         },
         browser: row.browser,
-        timestamp: new Date(row.timestamp),
+        timestamp: new Date(row.timestamp.slice(1, -1)),
```
Comment (jess-sys) [inline src/classes/Standardizer/plugins/Discord/getters/getConnections.ts:27]: Use standard Date.parse() method : https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Global_Objects/Date/parse

```suggestion
        timestamp: new Date(Date.parse(row.timestamp)),
```

**Review by jess-sys — CHANGES_REQUESTED** [review]

**Review by jess-sys — APPROVED** [review]
OK. The data originating from Discord is so poorly formatted.

**Comment by jess-sys** [comment]
Linked to https://youtrack.neutheria.org/issue/D4-17 (but does not resolves)

**Review by super-bunny — APPROVED** [review]
OK

**Pull request merged**